### PR TITLE
[ID] Update IRC

### PR DESCRIPTION
**Pull request by Fhoxydeo**
reupload from : https://github.com/ppy/osu-wiki/pull/5275

### CODE REVIEW

File: wiki/Internet_Relay_Chat/id.md [hunk 1]
```diff
@@ -1,80 +1,85 @@
----
-outdated: true
----
-
 # Internet Relay Chat
 
+*Untuk informasi lebih lanjut, lihat: [Internet Relay Chat (Wikipedia)](https://id.wikipedia.org/wiki/IRC "Wikipedia")*
+
 [Internet Relay Chat](https://id.wikipedia.org/wiki/IRC), dikenal juga sebagai IRC, adalah sebuah protokol standar yang sering digunakan untuk mengobrol dengan banyak pengguna lain yang tersedia.
```
Comment (mousesalt) [inline wiki/Internet_Relay_Chat/id.md:5]:
```suggestion
**Internet Relay Chat**, dikenal juga sebagai IRC, adalah sebuah protokol standar yang sering digunakan untuk mengobrol dengan banyak pengguna lain yang tersedia.
```

File: wiki/Internet_Relay_Chat/id.md [hunk 2]
```diff
@@ -1,80 +1,85 @@
...
 
 ## osu!Bancho
 
-osu!Bancho menyediakan gateway untuk mengakses IRC (obrolan dalam game). Anda dapat terhubung dengan klien anda sendiri dan mulai obrolan dengan orang lain tanpa harus menggunakan klien osu!. Mohon perhatikan bahwa prokotol IRC ini telah dimodifikasi oleh karena itu jangan harap fitur klien anda berfungsi dengan benar.
+*Catatan: server ini hanya menyediakan sebagian penerapan dari protokol IRC. beberapa fitur tak berdasar seperti [HexChat](https://hexchat.github.io/) mungkin tidak bekerja dengan baik atau tidak sama sekali.*
```
Comment (mousesalt) [inline wiki/Internet_Relay_Chat/id.md:9]:
```suggestion
*Catatan: server ini hanya menyediakan sebagian penerapan dari protokol IRC. Beberapa fitur pada client IRC yang tak berdasar seperti di [HexChat](https://hexchat.github.io) mungkin tidak bekerja dengan baik atau tidak sama sekali.*
```

File: wiki/Internet_Relay_Chat/id.md [hunk 3]
```diff
@@ -1,80 +1,85 @@
...
 
 - **Server:** `irc.ppy.sh`
-- **Port:** `6667` (standar)
-- **Username:** Nama pengguna osu! anda (ganti spasi dengan garis bawah)
+- **Username:** Nama pengguna osu! Anda, contoh (`beppy master 1000` menjadi `beppy_master_1000`)
+- **Port:** `6667` 
 - **Password:** Dapatkan di [Otentikasi IRC](https://osu.ppy.sh/p/irc).
 
-*Kata sandi IRC anda berbeda dengan kata sandi akun osu! anda. **Jangan dibagikan kepada orang lain**.*
+*Catatan: Kata sandi IRC Anda berbeda dengan kata sandi akun osu! Anda. **Jangan dibagikan kepada orang lain**.*
```
Comment (mousesalt) [inline wiki/Internet_Relay_Chat/id.md:22]:
```suggestion
*Peringatan: Kata sandi IRC Anda berbeda dengan kata sandi akun osu! Anda. ***Jangan dibagikan kepada orang lain***.
```

Comment (mousesalt) [reply wiki/Internet_Relay_Chat/id.md:22]: next time please using **warning** translation as bolding sentence instead

File: wiki/Internet_Relay_Chat/id.md [hunk 4]
```diff
@@ -1,80 +1,85 @@
...
 | `/ignore <username>` | Mengabaikan pengguna (menyembunyikan pesan mereka) |
+| `/away <message>` | Meninggalkan pesan untuk semua yang mencoba menghubungimu |
+| `/away` | Menghapus pesan tinggal |
+| `/query <username>` | Membuka kanal percakapan pribadi dengan pengguna lain (ganti spasi dengan tanda garis bawah) |
 
-## Menonaktifkan Pemberitahuan Masuk/Meninggalkan
+## Menonaktifkan pesan JOIN/PART/QUIT
 
-Mengetahui pemain yang masuk atau meninggalkan adalah hal baik, namun jika kanalnya sangat ramai seperti `#osu`, anda akan menerima pemberitahuan masuk/meninggalkan terus menerus dan tidak dapat mengikuti percakapan. Oleh karena itu, biasanya pesan-pesan ini lebih disarankan untuk disembunyikan.
+Pada dasarnya, beberapa klien IRC memberitahumu ketika pengguna bergabung atau pergi meninggalkan kanal. Untuk jaringan seperti osu!Bancho dengan ribuan pengguna yang sering bergabung dan pergi, pesan semacam itu menjadi sebuah spam, yang mungkin ingin Anda sembunyikan. Beberapa daftar klien IRC yang umum dan cara menonaktifkan pesan tersebut dijelaskan di bawah ini:
```
Comment (mousesalt) [inline wiki/Internet_Relay_Chat/id.md:38]:
```suggestion
Pada dasarnya, beberapa klien IRC memberitahumu ketika seorang pengguna bergabung atau meninggalkan kanal. Untuk jaringan seperti osu!Bancho dengan ribuan pengguna yang sering bergabung dan pergi, pesan semacam itu menjadi spam, yang mungkin ingin Anda sembunyikan. Beberapa daftar klien IRC yang umum dan cara menonaktifkan pesan tersebut dijelaskan di bawah ini:
```

File: wiki/Internet_Relay_Chat/id.md [hunk 5]
```diff
@@ -1,80 +1,85 @@
...
+| [ircII](http://www.eterna.com.au/ircii/ "ircII") | Ketik `IGNORE * CRAP` |
+| [Irssi](https://irssi.org "Irssi") | Ketik `/ignore * JOINS PARTS QUITS` |
+| [Weechat](https://weechat.org/ "Weechat") | Ketik `/filter add joinquit * irc_join,irc_part,irc_quit *` |
+| [KVIrc](https://www.kvirc.net/ "KVIrc") | (Merujuk ke [the KVIrc wiki page](https://github.com/kvirc/KVIrc/wiki/FAQ#how-do-i-suppress-join-part-and-quit-messages "GitHub")) |
+| [mIRC](https://www.mirc.com/ "mIRC") | Buka pengaturan mIRC (`Tools` -> `Options`, atau `Alt` + `O`), pilih `IRC`, tekan `Events...`  dan pilih `Hide` untuk bergabung dan pergi |
+| [Quassel IRC](https://quassel-irc.org/ "Quassel IRC") | Klik kanan di dalam chat dan pilih Joins/Parts/Quits di "Hide Events" menu |
+| [XChat](http://xchat.org/ "XChat") | Ketik `/set irc_conf_mode 1` |
 
-### Menonaktifkan Pemberitahuan Masuk/Meninggalkan Pada Klien IRC
+## Pertanyaan yang sering diajukan
```
Comment (mousesalt) [inline wiki/Internet_Relay_Chat/id.md:51]:
```suggestion
## Pertanyaan yang sering diajukan (FAQ)
```

Comment (mousesalt) [reply wiki/Internet_Relay_Chat/id.md:51]: we still using FAQ btw

File: wiki/Internet_Relay_Chat/id.md [hunk 6]
```diff
@@ -1,80 +1,85 @@
...
-| [ircII](http://www.eterna.com.au/ircii/) | Ketik `/ignore * crap` |
-| [Irssi](https://irssi.org) | Ketik `/ignore -channels #somechannel * JOINS PARTS QUITS` |
-| [Weechat](https://weechat.org/) | Ketik `/filter add irc_smart_weechat irc.username.#channel irc_smart_filter *`. **Catatan:** Ganti **username** dengan nama pengguna osu! anda. |
-| [KVIrc](https://www.kvirc.net/) | Kunjungi [utas ini](http://www.kvirc.ru/forum/?topic=609.0) di forum resmi KVIrc |
-| [mIRC](https://www.mirc.com/) | Buke Pengaturan mIRC (Tools - Options / Alt + O), di bawah IRC, klik tombol "Events..." dan ganti Joins and Parts menjadi "Hide" |
-| [Quassel IRC](https://quassel-irc.org/) | Klik kanan di jendela obrolan, lalu pilih Hide Events » Join/Part/Quit. |
-| [XChat](http://xchat.org/) | Ketik `/set irc_conf_mode 1` (atau [2](http://xchat.org/faq/#q211) untuk menonaktifkan pemberitahuan dari seluruh channel). |
+### Saya tidak bisa bergabung
+
+osu!Bancho menggunakan otentikasi teks biasa - pastikan tidak ada mode otentikasi khusus yang dipilih dalam pengaturan klien IRC Anda.
```
Comment (mousesalt) [inline wiki/Internet_Relay_Chat/id.md:55]:
```suggestion
osu!Bancho menggunakan otentikasi *plain text* - pastikan tidak ada mode otentikasi khusus yang dipilih dalam pengaturan klien IRC Anda.
```

Comment (mousesalt) [reply wiki/Internet_Relay_Chat/id.md:55]: dont even know what is teks biasa while commonly use to plain text.

File: wiki/Internet_Relay_Chat/id.md [hunk 7]
```diff
@@ -1,80 +1,85 @@
...
-## Pertanyaan yang Sering Diajukan
+### Saya mendapatkan kesalahan "Bad Authentication Token"
 
-### Saya mendapat kesalahan "Token Otentikasi Salah"
+Ikuti cara ini:
 
-1. Pastikan anda menggunakan kata sandi dari halaman [Otentikasi IRC](https://osu.ppy.sh/p/irc).
-2. Jika nama pengguna osu! anda memiliki spasi, ganti dengan garis bawah (misalnya **This Username** menjadi **This_Username**)
+1. Pastikan Anda menggunakan password yang sesuai yang didapatkan dari [halaman otentikasi IRC](https://osu.ppy.sh/p/irc).
+2. Jika nama penggunamu menggunakan spasi, gantilah dengan garis bawah (contoh: `This Username` menjadi `This_Username`).
```
Comment (mousesalt) [inline wiki/Internet_Relay_Chat/id.md:64]:
```suggestion
2. Jika nama pengguna Anda menggunakan spasi, gantilah dengan garis bawah (contoh: `Username ini` menjadi `Username_ini`).
```

File: wiki/Internet_Relay_Chat/id.md [hunk 8]
```diff
@@ -1,80 +1,85 @@
...
+### Bagaimana Saya menulis nama pengguna lain dengan cepat?
+
+Ketik beberapa kata pertama dari nama pengguna, lalu tekan `Tab` untuk memunculkan nama pengguna dengan lengkap secara otomatis.
+
+### Mengapa beberapa nama pengguna diawali dengan tanda yang berbeda?
+
+Konsep IRC pada umumnya memiliki beberapa mode kanal atau sekumpulan aksi di mana setiap pengguna dapat menggunakannya. Di osu!Bancho terdapat dua mode yang digunakan untuk mewakilkan kelompok khusus pengguna obrolan:
+
+- `+`, atau "voice status": pengguna terhubung dengan klien IRC eksternal
+- `@`, or "chat operator status": pengguna adalah seorang moderator ([GMT](/wiki/GMT) atau [NAT](/wiki/NAT))
```
Comment (mousesalt) [inline wiki/Internet_Relay_Chat/id.md:79]:
```suggestion
- `@`, atau "chat operator status": pengguna adalah seorang moderator ([GMT](/wiki/GMT) atau [NAT](/wiki/NAT))
```

**Review by mousesalt — COMMENTED** [review]
should be fine after this

File: wiki/Internet_Relay_Chat/id.md [hunk 9]
```diff
@@ -1,80 +1,85 @@
...
 
 ## osu!Bancho
 
-osu!Bancho menyediakan gateway untuk mengakses IRC (obrolan dalam game). Anda dapat terhubung dengan klien anda sendiri dan mulai obrolan dengan orang lain tanpa harus menggunakan klien osu!. Mohon perhatikan bahwa prokotol IRC ini telah dimodifikasi oleh karena itu jangan harap fitur klien anda berfungsi dengan benar.
+*Catatan: server ini hanya menyediakan sebagian penerapan dari protokol IRC. Beberapa fitur pada client IRC yang tak berdasar seperti di [HexChat](https://hexchat.github.io) mungkin tidak bekerja dengan baik atau tidak sama sekali.*
 
-**Pemberitahuan: [HexChat](https://hexchat.github.io/) diketahui memiliki masalah dengan IRC osu!** ([laporan bug dari GitHub HexChat](https://github.com/hexchat/hexchat/issues/818)), jika hal ini mengganggu anda, pertimbangkan untuk menggunakan klien lain .
+*osu!Bancho* (sering disingkat menjadi *Bancho*) menawarkan untuk mengakses IRC (in-game chat). Anda dapat menggunakan klien IRC apa pun untuk terhubung ke osu!Bancho (lihat daftar IRC di bawah yang paling umum digunakan).
```
Comment (wowcake) [inline wiki/Internet_Relay_Chat/id.md:11]:
```suggestion
*osu!Bancho* (sering disingkat menjadi *Bancho*) menawarkan untuk mengakses IRC (in-game chat). Anda dapat menggunakan klien IRC apapun untuk terhubung ke osu!Bancho (lihat daftar IRC di bawah yang paling umum digunakan).
```

Comment (wowcake) [reply wiki/Internet_Relay_Chat/id.md:11]: according to kkbi
https://kbbi.web.id/apapun

File: wiki/Internet_Relay_Chat/id.md [hunk 10]
```diff
@@ -1,80 +1,85 @@
...
 
-## Cara Menghubungkan
+## Cara menghubungkan
 
-Setelah Anda memiliki klien, Anda perlu mengatur pengaturan server.
+Setelah Anda memiliki klien, Anda perlu mengatur pengaturan server (Anda mungkin perlu untuk menambah server dahulu) .
```
Comment (wowcake) [inline wiki/Internet_Relay_Chat/id.md:15]:
```suggestion
Setelah Anda memiliki sebuah klien IRC, Anda perlu mengatur pengaturan server (Anda mungkin perlu untuk menambah server dahulu).
```

Comment (wowcake) [reply wiki/Internet_Relay_Chat/id.md:15]: make things clearer things a bit

File: wiki/Internet_Relay_Chat/id.md [hunk 11]
```diff
@@ -1,80 +1,85 @@
...
 
 - **Server:** `irc.ppy.sh`
-- **Port:** `6667` (standar)
-- **Username:** Nama pengguna osu! anda (ganti spasi dengan garis bawah)
+- **Username:** Nama pengguna osu! Anda, contoh (`beppy master 1000` menjadi `beppy_master_1000`)
```
Comment (wowcake) [inline wiki/Internet_Relay_Chat/id.md:18]:
```suggestion
- **Username:** Nama pengguna osu! Anda. (ganti spasi dengan garis bawah, contoh `beppy master 1000` menjadi `beppy_master_1000`)
```

Comment (wowcake) [reply wiki/Internet_Relay_Chat/id.md:18]: what happened to the part where it explains regarding name with underscore?

File: wiki/Internet_Relay_Chat/id.md [hunk 12]
```diff
@@ -1,80 +1,85 @@
...
+
+### Mengapa beberapa nama pengguna diawali dengan tanda yang berbeda?
+
+Konsep IRC pada umumnya memiliki beberapa mode kanal atau sekumpulan aksi di mana setiap pengguna dapat menggunakannya. Di osu!Bancho terdapat dua mode yang digunakan untuk mewakilkan kelompok khusus pengguna obrolan:
+
+- `+`, atau "voice status": pengguna terhubung dengan klien IRC eksternal
+- `@`, atau "chat operator status": pengguna adalah seorang moderator ([GMT](/wiki/GMT) atau [NAT](/wiki/NAT))
 
-### Apa fungsi status suara yang saya miliki ini? Saya juga melihat beberapa orang memilikinya
+Pengguna yang terhubung melalui klien osu! atau website tidak mempunyai prefix.
```
Comment (wowcake) [inline wiki/Internet_Relay_Chat/id.md:81]:
```suggestion
Pengguna yang terhubung melalui klien osu! atau website tidak akan mempunyai prefix.
```

Comment (wowcake) [reply wiki/Internet_Relay_Chat/id.md:81]: readers could misread it just as "website tidak mempunyai prefix"

File: wiki/Internet_Relay_Chat/id.md [hunk 13]
```diff
@@ -1,80 +1,85 @@
...
 
-Semua pengguna dengan *status suara* terhubung menggunakan klien IRC, kecuali moderator chat yang selalu memiliki status *operator (+o)* terlepas dari klien apapun yang mereka gunakan.
+### Seseorang mengirim pesan, tetapi mereka tidak ada dalam daftar pengguna kanal!
 
-Pengguna yang tidak memiliki status, terhubung menggunakan klien osu!.
+Mereka menggunakan [versi web chat](https://osu.ppy.sh/community/chat) atau terhubung via [osu!lazer](https://github.com/ppy/osu "GitHub").
```
Comment (wowcake) [inline wiki/Internet_Relay_Chat/id.md:85]:
```suggestion
Mereka menggunakan [versi web chat](https://osu.ppy.sh/community/chat) atau terhubung melalui klien [osu!lazer](https://github.com/ppy/osu "GitHub").
```

**Review by wowcake — CHANGES_REQUESTED** [review]

File: wiki/Internet_Relay_Chat/id.md [hunk 14]
```diff
@@ -8,14 +8,14 @@
 
 *Catatan: server ini hanya menyediakan sebagian penerapan dari protokol IRC. Beberapa fitur pada client IRC yang tak berdasar seperti di [HexChat](https://hexchat.github.io) mungkin tidak bekerja dengan baik atau tidak sama sekali.*
 
-*osu!Bancho* (sering disingkat menjadi *Bancho*) menawarkan untuk mengakses IRC (in-game chat). Anda dapat menggunakan klien IRC apa pun untuk terhubung ke osu!Bancho (lihat daftar IRC di bawah yang paling umum digunakan).
```
Comment (mousesalt) [inline wiki/Internet_Relay_Chat/id.md]: this actually correct grammar, no need to change it.

**Review by mousesalt — APPROVED** [review]
overall looks good :+1

**Review by TicClick — APPROVED** [review]

**Pull request merged**